charts/operator/deployment.go

```diff
@@ -27,6 +28,24 @@ import (
 	"k8s.io/utils/ptr"
 )
 
+const (
+	// Injected bound service account token expiration which triggers monitoring of its time-bound feature.
+	// Reference
+	// https://github.com/kubernetes/kubernetes/blob/ae53151cb4e6fbba8bb78a2ef0b48a7c32a0a067/pkg/serviceaccount/claims.go#L38-L39
+	WarnOnlyBoundTokenExpirationSeconds = 60*60 + 7
```

Comment:
nit: Give this a name that's meaningful to this package. `WarnOnlyBound` doesn't really make sense here IMO.

Also no need for this to be exported :)

Reply:
Changed.